### DISCUSSION TITLE
Phidgets says my Linux Mint 21.3 Virginia system isn't supported?

### DISCUSSION THREAD

**Discussion by tomhusband** [artisan-roaster-scope/artisan#1518] [Q&A] [answered]
I installed Linux Mint on a Lenovo T480 laptop and when I try to install Phidgets 22 libraries using the install scripts [here](https://www.phidgets.com/docs/OS_-_Linux#Non-Root-1) however I get a terminal message saying my distribution,Virginia, isn't supported. I had Phidgets running on my old laptop with this version of Linux so I'm puzzled. Any suggestions?

**Comment by roasterdave** [comment, marked as answer]
Tom, I don't know if there is someone with the knowledge to help here.  Maybe an answer will come along.  However, since these drivers are outside Artisan's control I recommend reaching out to Phidgets support directly.  Please let us know what you learn.

**Reply by tomhusband** [reply]
Thanks Dave. I realized that this morning and will see what I can find out from them directly.

**Reply by tomhusband** [reply]
Looks like Phidget [updated](https://www.phidgets.com/phorum/viewtopic.php?t=10796) the script to make it future proof. I'll give it a try.
